vitest.shared.config.mts

```diff
@@ -17,6 +17,7 @@ export default defineConfig({
   esbuild: { target: "es2022" },
   test: {
     globalSetup: ["vitest.global-setup.ts"],
+    setupFiles: ["./vitest.setup-files.ts"],
```

Comment:
As `setupFiles` runs [before every test](https://vitest.dev/config/#setupfiles), could this be put in `globalSetup` to only run once?

Reply:
`expect` is not defined in the global setup, which isn't ideal.